examples/speak/websocket/simple/Program.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright 2024 Deepgram .NET SDK contributors. All Rights Reserved.
+// Use of this source code is governed by a MIT license that can be found in the LICENSE file.
+// SPDX-License-Identifier: MIT
+
+using Deepgram.Models.Authenticate.v1;
+using Deepgram.Models.Speak.v1.WebSocket;
+using Deepgram.Logger;
+
+
+namespace SampleApp
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            // Initialize Library with default logging
+            // Normal logging is "Info" level
+            //Library.Initialize();
+            // OR very chatty logging
+            Library.Initialize(LogLevel.Debug); // LogLevel.Default, LogLevel.Debug, LogLevel.Verbose
+
+            Console.WriteLine("\n\nPress any key to stop and exit...\n\n\n");
+
+            // use the client factory with a API Key set with the "DEEPGRAM_API_KEY" environment variable
+            var clientOptions = new DeepgramWsClientOptions("", "api.beta.deepgram.com");
```

Comment:
beta url might need to be removed

Reply:
will remove when I update all the examples